content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -673,8 +674,6 @@ Each feature gate is designed for enabling/disabling a specific feature:
   which allows a cluster operator to live migrate
   controllers from the kube-controller-manager into an external controller-manager
   (e.g. the cloud-controller-manager) in an HA cluster without downtime.
-- `CPUManager`: Enable container level CPU affinity support, see
-  [CPU Management Policies](/docs/tasks/administer-cluster/cpu-management-policies/).
```

Comment:
Don't remove this text; it's still needed.

Reply:
ah, sorry. Restoring.